moveit_setup_assistant/templates/moveit_config_pkg_template/package.xml.template

```diff
@@ -20,6 +20,7 @@
   <run_depend>joint_state_publisher</run_depend>
   <run_depend>robot_state_publisher</run_depend>
   <run_depend>xacro</run_depend>
+  <run_depend>warehouse_ros_mongo</run_depend>
```

Comment:
All moveit config packages have depended on mongo in the past, so
continuing that isn't a big deal assuming the mongo dependencies are easy
to install and well maintained

On Aug 19, 2016 5:49 AM, "Michael Görner" <notifications@github.com> wrote:

> In moveit_setup_assistant/templates/moveit_config_pkg_
> template/package.xml.template
> <https://github.com/ros-planning/moveit/pull/81#discussion_r75468584>:
>
> > @@ -20,6 +20,7 @@
> >    <run_depend>joint_state_publisher</run_depend>
> >    <run_depend>robot_state_publisher</run_depend>
> >    <run_depend>xacro</run_depend>
> > +  <run_depend>warehouse_ros_mongo</run_depend>
>
> Yes, that is a problem. We probably shouldn't make *all* `moveit_config`s
> depend on warehouse_ros_mongo, out of the box. The simplest way to remove
> this is to add the changes in comments, so the user can uncomment them if
> he needs it. Not sure if that's what we want to do though. Does anyone have
> an opinion on this?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ros-planning/moveit/pull/81/files/f21296943161b6a702259cb183ecc81e8b0063b2#r75468584>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAiPpH8CE4RVY_vrhA5yjxNjN4qdwcnMks5qhZhTgaJpZM4JoY-r>
> .
>